guides/common/modules/proc_configuring-your-deployment-to-run-ansible-roles.adoc

```diff
@@ -0,0 +1,39 @@
+[id="configuring-your-deployment-to-run-Ansible-roles_{context}"]
+
+= Configuring your Deployment to Run Ansible Roles
+
+ifeval::["{build}" == "satellite"]
+In {Project}, you can import Ansible roles to help with automation of routine tasks. Ansible is enabled by default only on {Project}.
+endif::[]
+
+ifeval::["{build}" != "satellite"]
+In {Project}, you can import Ansible roles to help with automation of routine tasks. To enable the Ansible plugin in {Project}, enter the following command:
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+  # {foreman-installer} --enable-foreman-plugin-ansible \
+  --enable-foreman-proxy-plugin-ansible
+----
+
+endif::[]
+
+
+Complete this procedure to configure your {Project} deployment to run Ansible roles.
+
+.Procedure
+
+. Add the roles to the `/etc/ansible/roles` directory on the {Project} and all {SmartProxies} from where you want to use the roles. If you want to use custom or third party Ansible roles, ensure to configure an external version control system to synchronize roles between {Project} and {SmartProxies}.
+
+. On all {SmartProxies} that you want to use to run Ansible roles on hosts, enable the Ansible plug-in:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {installer-scenario-smartproxy} \
+--enable-foreman-proxy-plugin-ansible true
+----
+
+. Distribute SSH keys to enable {SmartProxies} to connect to hosts using SSH. For more information, see {BaseURL}managing_hosts/running_jobs_on_hosts#sect-Managing_Hosts-Establishing_a_Secure_Connection_for_Remote_Commands[Distributing SSH Keys for Remote Execution] in the _Managing Hosts_ guide. {Project} runs Ansible roles the same way it runs remote execution jobs.
+
+. Import the Ansible roles into {Project}.
+
```

Comment:
Sounds like the step to assign Ansible roles to hosts is missing here.

Reply:
None of these are my changes. 
I just am organizing the files. 
These are live errors. 
Please let's do this on another PR.